### PR TITLE
test/cases/ubi-wsl: public ip fixes

### PR DESCRIPTION
**Pull request by croissanne**
Use the "Standard" public ip sku, as "Basic" produces warnings.

Spin until the vm gets assigned a valid ipv4 address.

### CODE REVIEW

**Review by ondrejbudai — DISMISSED** [review]

**Review by thozza — APPROVED** [review]

**Comment by croissanne** [comment]
Looking at the test, the retry counter for the host went all the way up to 9 (out of 10). Maybe more tries would be prudent ._.

**Review by thozza — APPROVED** [review]
:+1: 

Just realized that increasing the `sleep` would be also OK :innocent:

File: test/cases/ubi-wsl.sh [hunk 1]
```diff
@@ -192,20 +192,33 @@ $AZURE_CMD vm create \
...
+    HOST=$($AZURE_CMD vm list-ip-addresses \
+       --resource-group "$AZURE_RESOURCE_GROUP" \
+       --name "wsl-vm-$TEST_ID" \
+       --query "[].virtualMachine.network.publicIpAddresses[0].ipAddress" \
+       --output tsv)
+
+    if echo "$HOST" | grep -Eq "^([0-9]{1,3}[\.]){3}[0-9]{1,3}$"; then
+        break
+    fi
+    if [ "$LOOP_COUNTER" = "10" ]; then
```
Comment (bcl) [inline test/cases/ubi-wsl.sh:214]: The counter is never going to go past 10 :)

Comment (thozza) [reply test/cases/ubi-wsl.sh:214]: :facepalm:

Comment (croissanne) [reply test/cases/ubi-wsl.sh:214]: I'm an idiot :crying_cat_face:

**Pull request merged**